partitions.go

```diff
@@ -231,6 +253,12 @@ func (p *partitions) getPeers(partition int) []string {
 
 	peers := make([]string, len(p.remote[partition]))
 	copy(peers, p.remote[partition])
+
+	oldPeers := make([]string, 1024)
+	copy(oldPeers, p.old[partition])
+	// Append old peers to peer list, in case of Zookeeper issues.
```

Comment:
Again, I don't think this comment tells the reader much (if they don't already know what's up).